system-tests/project-fixtures/svelte/src/mount.cy.js

```diff
@@ -0,0 +1,59 @@
+import Counter from "./Counter.svelte";
+import Context from "./Context.svelte";
+import Store from "./Store.svelte";
+import { messageStore } from "./store";
+
+describe("Svelte mount", () => {
+  it("mounts and unmounts", () => {
+    cy.mount(Counter).then(({ unmount }) => {
+      cy.contains("h1", "Count is 0");
+      unmount();
+      cy.contains("h1", "Count is 0").should("not.exist");
+    });
+  });
+
+  it("reacts to state changes", () => {
+    cy.mount(Counter);
+    cy.contains("h1", "Count is 0");
+    cy.get("button").click();
+    cy.contains("h1", "Count is 1");
+  });
+
+  it("accepts props", () => {
+    cy.mount(Counter, { props: { count: 42 } });
+    cy.contains("h1", "Count is 42");
+  });
+
+  it("accepts context", () => {
+    const payload = { msg: "This value came from context!" };
+    const context = new Map();
+    context.set("myKey", payload);
+
+    cy.mount(Context, { context });
+    cy.contains("h1", payload.msg);
+  });
+
+  it("spies on outputs", () => {
+    cy.mount(Counter).then(({ component }) => {
+      component.$on("change", cy.spy().as("changeSpy"));
+      cy.get("button").click();
+      cy.get("@changeSpy").should("have.been.called");
```

Comment:
Here's a slightly less verbose version of what we talked about for confirming the value of the event:

```suggestion
      cy.get("@changeSpy").should("have.been.called")
      .and('satisfy', ({args}) => {
        const [{detail}] = args[0]
        return Cypress._.isEqual(detail, {count: 0})
      });
```

Still clunky. Agree that it would nice to get a simple pattern going for this.